docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Restore
+val sourcePrefix = "/mnt/overwatch_playground/721-snap/full"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/restore"
+Restore.isValid(sourcePrefix,targetPrefix)
+```
+
+Upon running the above function you will get proper output whether the input parameters you would be using for Restore process,passed the validation check or not. One example of the same is as below:
+
+```
+SourcePrefix Path Exists.
+Target Path /mnt/overwatch_playground/721-snap/restore is Empty and is Suitable for Restore Process
+Validation successful.You Can proceed with Restoration process
+import com.databricks.labs.overwatch.pipeline.Restore
+res43: Boolean = true
+```
+
+
+### Migration Process
+Migration Process is same as Snapshot process with some added Functionality. Below are the steps involved in Migration Process:
+
+1. Stop Overwatch jobs  (Need to be done by User)
+2. Migrate all old data to new location. (Would be done by OW Migration Process) 
+3. Update Config with new storage location (Would be done by OW Migration Process)
+4. Delete Old Database  (Would be done by OW Migration Process)
+5. Resume Overwatch jobs (Need to be done by User)
+
+##### Migration through Databricks Notebook
+The code snippet to run the Migration process is as below:
+```scala
+import com.databricks.labs.overwatch.pipeline.Migration
+val sourceETLDB = "ow_bronze_migration_etl"
+val migrateRootPath = "/mnt/overwatch_playground/721-snap/migration"
+val configPath = "abfss://overwatch-field-playground@overwatchglobalinternal.dfs.core.windows.net/sourav/configs/721-migration"
+val tablesToExclude = "notebook_silver:notebook_gold"
+Migration.process(sourceETLDB,migrateRootPath,configPath)
+```
+We need below parameters to run the Migration process through Databricks Notebook:
+
+| Param           | Type     | Optional   | Description                                                                                                                           |
+|-----------------|----------|------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| sourceETLDB     | String   | No         | Source Database Name for which Migration need to be done.                                                                             |
+| migrateRootPath | String   | No         | Target path to where migration need to be performed.                                                                                  |
+| configPath      | String   | No         | Configuration Path where the config file for the source is present. Path can be CSV file or delta path ot delta table.                |
+| tablesToExclude | String   | Yes        | Array of table names to exclude from the snapshot. This is the table name only - without the database prefix. By Default it is empty. |
```

Comment:
Change to: "Array of table names to exclude from the migration. This is the table name only - without the database prefix. By Default it is empty."

Reply:
Done